aws/inspect.go

```diff
@@ -78,7 +78,7 @@ func HandleResourceTypeSelections(
 	return resourceTypes, nil
 }
 
-func InspectResources(q *Query) (*AwsAccountResources, error) {
+func InspectResources(q *Query, listUnaliasedKMSKeys bool) (*AwsAccountResources, error) {
```

Comment:
Is there any way to smuggle this boolean in a new `Query` field instead of making a new argument to `InspectResources`? Wondering if that would be easier to maintain going forward or not.